docarray/documents/audio.py

```diff
@@ -94,11 +96,30 @@ class MultiModalDoc(BaseDoc):
     ```
     """
 
-    url: Optional[AudioUrl] = None
-    tensor: Optional[AudioTensor] = None
-    embedding: Optional[AnyEmbedding] = None
-    bytes_: Optional[AudioBytes] = None
-    frame_rate: Optional[int] = None
+    url: Optional[AudioUrl] = Field(
+        description='The url to a (potentially remote) audio file that can be loaded',
+        example='https://github.com/docarray/docarray/blob/main/tests/toydata/hello.mp3?raw=true',
+        default=None,
+    )
+    tensor: Optional[AudioTensor] = Field(
+        description='''Tensor object of the audio which can be specified to one of
+        `AudioNdArray`, `AudioTorchTensor`, `AudioTensorFlowTensor`''',
+        default=None,
+    )
+    embedding: Optional[AnyEmbedding] = Field(
+        description='Store an embedding: a vector representation of the audio.',
+        example='[0, 1, 0]',
```

Comment:
```suggestion
        example=[0, 1, 0],
```

Reply:
Will change this. Should've gone through the docs for pydantic